### PR TITLE
Manually Migrate Sites 

### LINKED ISSUE #1863: Migrate Sites to Pantheon: Manual Method Doc Update
Re: [Migrate Sites to Pantheon: Manual Method](https://pantheon.io/docs/migrate-manual)
## Issue Description:

There are now only 2 options for import for WordPress sites: Url for existing site or manual migration. Sites that are not yet LIVE therefore only have the option of manual migration (no archive upload anymore).

## Suggested Resolution 
In this section:
"Manually migrate your site to Pantheon when any of the following apply:"

We should add a bullet point with something like:
- It is a WordPress site and it is not yet live on a url / only exists on local machine.

### PR DESCRIPTION
**Pull request by nataliejeremy**
Closes #1863 and #1862 

## Effect
PR includes the following changes:
- adds local WP site to the list of reasons to migrate manually

Thanks for creating the issue @tinefin 
@rachelwhitton please review and merge

### CODE REVIEW

**Comment by rachelwhitton** [comment]
Might be better to recommend Terminus import as an option first, then suggest manual migration for larger archives. Related: https://github.com/pantheon-systems/documentation/issues/1862#issuecomment-244966113

**Comment by ari-gold** [comment]
How about adding a Question/Answer at bottom doc? 

Question: How do I migrate a site from my local machine to Pantheon?
Answer: what Rachel propsed

**Comment by rachelwhitton** [comment]
+1 Thanks @nataliejeremy!

File: source/_docs/migrate-manual.md [hunk 1]
```diff
@@ -128,6 +128,16 @@ You should now have all three of the major components of your site imported into
 
 If something didn't work and you'd like to start over, you can cancel the site migration, delete the site, and try again. We also recommend making use of our **[Backups](/docs/create-backups/)** tool periodically during migration so you can restore to a known good state of your site.
 
+## Frequently Asked Questions (FAQs)
+
+#### How do I migrate a local site to Pantheon?
+You can import a WordPress site archive via URL (within file size limits) using [Terminus](/docs/terminus):
+
+```bash
+terminus sites import —url=<url>
```
Comment (ari-gold) [inline source/_docs/migrate-manual.md]: sites >> site

**Pull request merged**